src/Navigation.vue

```diff
@@ -97,11 +97,13 @@ export default {
 .VueCarousel-navigation-next {
   right: 0;
   transform: translateY(-50%) translateX(100%);
+  font-family: "Segoe UI Symbol";
```

Comment:
This is a good catch, but I think we should use a full system font stack here for the fallback -> https://css-tricks.com/snippets/css/system-font-stack/

Reply:
Thank you for your review. System font stack was added in a recent commit several minutes ago.